vsintegration/src/FSharp.UIResources/Strings.resx

```diff
@@ -165,4 +165,16 @@
   <data name="Unused_opens_code_fix" xml:space="preserve">
     <value>Remove unused open statements</value>
   </data>
+  <data name="Code_Lens" xml:space="preserve">
+    <value>Code-Lens</value>
```

Comment:
`CodeLens` is the official name

Reply:
Well IDK whether the writing is fine like this. I personally see its name consisting of two words.
@cartemp what is the naming convention?

Reply:
It's definitely not hyphenated: https://msdn.microsoft.com/en-us/library/dn269218.aspx